content/es/docs/getting-started/demos-talks-posts/talks.md

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
This is a live page that is being updated continuously. Can someone commit to keeping it always up to date? If not, it's better to leave it in english.

Reply:
I recommend not translating this particular page.

Reply:
Sounds good for me. One question, whats happens with the `es` folder structure?
- should I remove the file completely? Will not appear in the navigation bar.
- should I keep some reference to the generated page in the `en` folder?

Reply:
@vfarcic @sharepointoscar ?